plugins/vpc-bridge/network/bridge_windows.go

```diff
@@ -371,15 +428,45 @@ func (nb *BridgeBuilder) attachEndpointV2(ep *hcsshim.HNSEndpoint, netNSName str
 	return err
 }
 
+func (nb *BridgeBuilder) addNetworkPolicy(
+	nw *hcn.HostComputeNetwork,
+	policyType hcn.NetworkPolicyType,
+	policySettings interface{},
+) error {
```

Comment:
The function definition convention in this repo seems to be to declare the return type after the last parameter. Multiple occurrences in this file.

Reply:
I'm not totally clear about this one.
Since there were multiple params, I have moved each of them to a new line.

Reply:
I meant other functions append the return type to the end of the last parameter:
policySettings interface{}) error {